//python -m grpc_tools.protoc -I . --python_out=. --grpc_python_out=. ring.proto

syntax = "proto3";
package tic_tac_toe;

service RingElection {
    rpc StartElection(RingMessage) returns (RingMessage) {}
}

message RingMessage {
    int32 origin = 1;
    int32 max_id = 2;
    int32 rounds = 3;
    int32 leader = 4;
}